server.py

```diff
@@ -12,10 +12,11 @@ def create_application():
     """Create Flask application instance."""
     app = Flask(__name__)
     app.config['AI_SERVICE'] = os.environ.get('AI_SERVICE')
-    app.config['NEXT_MICROSERVICE_HOST'] = \
-        os.environ.get('NEXT_MICROSERVICE_HOST')
-    app.config['IF_NUM_TREES'] = int(os.environ.get('IF_NUM_TREES') or '0')
-    app.config['IF_SAMPLE_SIZE'] = int(os.environ.get('IF_SAMPLE_SIZE') or '0')
+    app.config['NEXT_SERVICE_URL'] = os.environ.get('NEXT_SERVICE_URL')
+    app.config['IF_NUM_TREES_FACTOR'] = \
+        float(os.environ.get('IF_NUM_TREES_FACTOR') or '0.2')
```

Comment:
Nitpick: this should be `float(is.environ.get('IF_NUM_THEES_FACTOR', .2))`. The `get` method can provide default value for you. No need for `or`. Please fix this it this expression and in the one below

Reply:
done